modules/nw-sriov-hwol-configuring-machine-config-pool.adoc

```diff
@@ -31,7 +31,7 @@ spec:
       - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,mcp-offloading <1>]}
   nodeSelector:
     matchLabels:
-      node-role.kubernetes.io/mcp-offloading <2>: ""
+      node-role.kubernetes.io/mcp-offloading <2>: "" <2>
```

Comment:
@ctauchen Can you review and let me know if this is the right line for the callout 2 mentioned below?

Reply:
So now there are two <2>, and there's still a non-styled <2> in the example code. Not sure if you want that first one there. The 2nd one is resolving as a callout, which seems to be what you're expecting.

Reply:
Ooo good catch, I missed that <2> in there. I'll remove that first one. Thanks!

Reply:
Now that you mention it, I see a <1> on line 31 that I think is supposed to be a callout. @ctauchen I moved that callout too if you can please take a look at both of these lines.

Reply:
@bergerhoffer Yes, that's the right line.

Reply:
Thanks! I'll go ahead and merge this in then.